vector/build.gradle

```diff
@@ -337,6 +337,9 @@ android {
 }
 
 dependencies {
+    implementation fileTree(dir: "libs", include: ["*.aar", "*.jar"])
```

Comment:
Is it necessary to add this line? I prefer to add explicit dependencies one by one if any

Reply:
No, sorry, this was added to test some pre-built libs and I completely forgot to delete it.